horace_core/sqw/@sqw/private/cut_accumulate_data_.m

```diff
@@ -50,7 +50,12 @@
 %sproj = obj.data.get_projection(header_av);
 sproj = obj.data.proj;
 saxes = obj.data.axes;
-[bloc_starts, block_sizes] = sproj.get_nrange(obj.data.npix,saxes,targ_axes,targ_proj);
+if numel(obj.data.npix) == 1 % single bin original grid
+    bloc_starts = 1;
+    block_sizes = obj.data.npix;
+else
+    [bloc_starts, block_sizes] = sproj.get_nrange(obj.data.npix,saxes,targ_axes,targ_proj);
+end
```

Comment:
is the single bin if just away of speeding up the calc or is there a gotcha in how get_nrange does things

Reply:
it was a bug in `get_nrange` causing problem with single  bin. I believe its fixed now, but if is still useful as `get_nrange` is doing a lot of work especially for highly non-commensurate lattices. Its useful performance improvement and a simple one